HW_1/zadanie1.py

```diff
@@ -0,0 +1,13 @@
+#Задача 2: Найдите сумму цифр трехзначного числа.
+
+#*Пример:*
+
+#123 -> 6 (1 + 2 + 3)
+#100 -> 1 (1 + 0 + 0) |
+n1 = 1;n2 = 2 ;n3 = 3 ;
```

Comment:
Такие записи не приняты в Python. Забудьте про `;`
Нудно так
```suggestion
n1 = 1
n2 = 2
n3 = 3
```
Или так
```suggestion
n1, n2, n3 =1, 2, 3
```